CHANGELOG.md

```diff
@@ -3,6 +3,92 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.11.0
+
+_Aug 3, 2023_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add new package `@mui/x-tree-view` containing the tree view component
```

Comment:
@richbustos this is important in a way that if someone is use that package from the Lab they will no longer get updates and will need to use the one from mui-x instead ;)

Reply:
Not yet
For now the lab still works and the core doc is still available.
I'll add a message in the core CHANGELOG when we make the lab `TreeView` just a re-export of ours
And another message when we completely remove it from the lab.

Reply:
The main PR is here: https://github.com/mui/material-ui/pull/38261

Reply:
> completely remove it from the lab.

This should be done in a major release, right?